app/Repositories/Eloquents/UserRepository.php

```diff
@@ -59,7 +59,9 @@ public function update($id, $request)
         }
         $data['name'] = $request->name;
         $data['email'] = $request->email;
-        $data['password'] = bcrypt($request->password);
+        if ($request->password != "") {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/UserRepository.php</br>**Line**: 62</br>String "" does not require double quotes; use single quotes instead